pkg/scheduler/core/spreadconstraint/group_clusters.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package spreadconstraint
 
 import (
+	"k8s.io/utils/ptr"
```

Comment:
Worth noting - the linter was failing because the original import (`k8s.io/utils/pointer`) is deprecated. This new import will work for newer karmada versions, but is not currently compatible with the version we use (v1.9.0) because it uses an older k8s-util version.

Reply:
Thanks for the information. 
I wonder if you maintain some private commits based on v1.9.0? 
I'm asking because I want to know if this going to be a problem for you. I mean if this patch not present on release 1.9.

Reply:
We have a couple private commits on our v1.9.0 branch, but most of them we are planning to contribute via our published proposals. I think we can try updating to the v1.10.0 release branch which will have the updated k8s-util version and can pick up this commit. I'll test v1.10 with out setup and confirm.

Reply:
Confirmed changes work with v1.10 - I think we can move forward with upgrading our local Karmada version.